concepts/05 Widgets/NumberBox/20 Keyboard Support.md

```diff
@@ -65,6 +65,59 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxNumberBox @initialized="registerKeyHandlers"
```

Comment:
We usually call `registerKeyHandler` on `mounted`/`componentDidMount()`:
https://js.devexpress.com/Documentation/20_1/Guide/Widgets/Button/Keyboard_Support/